BioSig Correction

hassan-albalawi opened the pull request:
The toolbox is actually using a function called 'sload' that can be
imported directly from BioSig. EEGLAB doesn't have this function; but it
has a BioSig Plugin that you must install if you're going to go with
EEGLAB.

anasimtiaz commented:
Thanks. Previous versions of EEGLAB came with the sload function but the newer version doesn't and requires BioSig plugin.

hassan-albalawi committed:
BioSig Correction

The toolbox is actually using a function called 'sload' that can be
imported directly from BioSig. EEGLAB doesn't have this function; but it
has a BioSig Plugin that you must install if you're going to go with
EEGLAB.
